### PR TITLE
feat: support the simplest influxql raw query

### PR DESCRIPTION
**Pull request by Rachelint**
## Which issue does this PR close?

Closes #
Part of #698 

## Rationale for this change
After #683, we has prepared enough for supporting to do some simplest query by influxql.
In this pr, we add the support above. 

## What changes are included in this PR?
+ Support basic statement converting(to sql statement) after #683
+ Add influxql http interface for testing.  

## Are there any user-facing changes?
User can do some simple query by influxql now.

## How does this change test
Test by ut and manually.

### CODE REVIEW

File: sql/src/influxql/util.rs [hunk 1]
```diff
@@ -76,6 +79,20 @@ pub fn parse_regex(re: &Regex) -> std::result::Result<regex::Regex, regex::Error
     regex::Regex::new(&pattern)
 }
 
+// Copy from influxql_iox.
+lazy_static! {
+    pub(crate) static ref SCALAR_MATH_FUNCTIONS: HashSet<&'static str> = HashSet::from([
```
Comment (ShiKaiWi) [inline sql/src/influxql/util.rs:84]: Seems no need to make it public?

File: sql/src/influxql/select/converter.rs [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+                    alias: Ident::new(alias.to_string()),
+                },
+                None => SelectItem::UnnamedExpr(expr),
+            })
+            .collect();
+
+        ensure!(
+            stmt.from.len() == 1,
+            Unimplemented {
+                msg: "from multiple measurements",
```
Comment (ShiKaiWi) [inline sql/src/influxql/select/converter.rs:78]:
```suggestion
                msg: "select from multiple measurements",
```

File: sql/src/influxql/select/converter.rs [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+                MeasurementName::Regex(re) => {
+                    return Convert {
+                        msg: format!("convert from to sql statement encounter regex, regex:{re}"),
+                    }
+                    .fail()
+                }
+            },
+            MeasurementSelection::Subquery(_) => {
+                return Unimplemented {
+                    msg: "from subquery",
```
Comment (ShiKaiWi) [inline sql/src/influxql/select/converter.rs:93]:
```suggestion
                    msg: "select from subquery",
```

File: sql/src/influxql/test_util.rs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+//! Test utils
+
+use common_util::error::GenericResult;
+use datafusion::sql::TableReference;
+use influxdb_influxql_parser::{parse_statements, select::SelectStatement, statement::Statement};
+
+use super::{planner::MeasurementProvider, select::rewriter::Rewriter};
+use crate::{provider::MetaProvider, tests::MockMetaProvider};
+
+impl MeasurementProvider for MockMetaProvider {
```
[the remark this answers is not in the document]
Comment (Rachelint) [reply sql/src/influxql/test_util.rs:12]: It need to be imported by multiple mods for testing when we implement other statements(such as show tag keys).

File: sql/src/influxql/select/converter.rs [hunk 5]
```diff
@@ -0,0 +1,388 @@
...
+                    Dimension::Wildcard => Convert {
+                        msg: "convert group by to sql statement encounter wildcard",
+                    }
+                    .fail(),
+                })
+                .collect::<Result<Vec<_>>>()?,
+            None => Vec::default(),
+        };
+
+        let body = Select {
```
[the remark this answers is not in the document]
Comment (Rachelint) [reply sql/src/influxql/select/converter.rs:138]: Yes...

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**